src/cluster.c

```diff
@@ -4589,9 +4671,13 @@ void clusterCommand(client *c) {
 "    Connect nodes into a working cluster.",
 "MYID",
 "    Return the node id.",
+"MYNAME",
+"    Return the human readable node name.",
+"SETNAME <human-readable-name>",
+"    Sets the human-readable node name.",
```

Comment:
My guess is this makes more sense as a config, since you'll want to set it at startup.